pkg/controller/certificaterequests/venafi/venafi_test.go

```diff
@@ -555,6 +555,7 @@ func TestSign(t *testing.T) {
 								LastTransitionTime: &metaFixedClockStart,
 							}),
 							gen.SetCertificateRequestCertificate(certPEM),
+							gen.SetCertificateRequestCA(certPEM),
```

Comment:
These tests use a self-signed cert so the CA *is* the cert.
I considered making a proper chain to use in the tests, but wasn't sure it was necessary to test here, since we test it in the E2E tests anyway.